Kudu.Services.Web/_Layout.cshtml

```diff
@@ -42,7 +54,7 @@
             </div>
             <div class="collapse navbar-collapse">
                 <ul class="nav navbar-nav">
-                    <li><a href="~/Env">Environment</a></li>
+                    <li><a href="~/Env.cshtml">Environment</a></li>
```

Comment:
Why doesn't /env just work? That's a small regression on Windows if we force the extension.